### PR TITLE
Update wording on WAN fed and intermediate_pki_path

### PR DESCRIPTION
**Pull request by kisunji**
Follow-up to https://github.com/hashicorp/consul/pull/17831

### CODE REVIEW

File: website/content/docs/connect/ca/vault.mdx [hunk 1]
```diff
@@ -139,8 +139,10 @@ The key after the slash refers to the corresponding option name in the agent con
   path does not exist, Consul will attempt to mount and configure this
   automatically.
 
-  When WAN Federation is enabled, every secondary
-  datacenter must specify a unique `intermediate_pki_path`.
+  When WAN Federation is enabled, every secondary datacenter which shares a common Vault cluster
+  must specify a unique `intermediate_pki_path`. If a Vault cluster is not being used by more than
+  one Consul datacenter, then this is not required. However, it is still recommended to use a
+  unique `intermediate_pki_path` per datacenter for operational and diagnostic clarity.
```
Comment (trujillo-adam) [inline website/content/docs/connect/ca/vault.mdx:145]:
```suggestion
  When WAN federation is enabled, every secondary datacenter that shares a common Vault cluster
  must specify a unique `intermediate_pki_path`. If a Vault cluster is not used by more than
  one Consul datacenter, then the intermediate PKI path setting is not required. We still recommend using a
  unique `intermediate_pki_path` for each datacenter, however, to improve operational and diagnostic clarity.
```

**Review by trujillo-adam — APPROVED** [review]
Thanks for fixing this. Just a few addition suggestions.

File: website/content/docs/connect/ca/vault.mdx (hunk 1 above)
Comment (kisunji) [reply website/content/docs/connect/ca/vault.mdx:145]: > If a Vault cluster is not used by more than one Consul datacenter, then the intermediate PKI path setting is not required.

The phrase above is slightly wrong; how does this read?

> If a Vault cluster is not used by more than one Consul datacenter, then **the uniqueness** is not required.

Comment (trujillo-adam) [reply website/content/docs/connect/ca/vault.mdx:145]: It's still kind of awkward. How about:

> If a Vault cluster is not used by more than one Consul datacenter, then you do not need to specify a unique value for the `intermediate_pki_path`. 

Right? You're saying that you still need to configure the setting, but the value doesn't have to be unique, correct?

Comment (kisunji) [reply website/content/docs/connect/ca/vault.mdx:145]: Yeah that works!

**Pull request merged**